### ISSUE TITLE
The Infested Protectors

### ISSUE THREAD

**Issue by Bloobeary** [Helios-Empire/Helios-Tracker#181]
Kind of bug:
The problem is with this quest: http://db.hellground.net/?quest=10896

What bug do you encounter?:
Killing the Infested Root-Walkers ( http://db.hellground.net/?npc=22095 ) should spawn 4-5 Wood Mites ( http://db.hellground.net/?npc=22419 ) and killing Rotting Forest-Ragers ( http://db.hellground.net/?npc=22307 ) should spawn 6-9 Wood Mites (the spawn chance is not 100%), but it's only one Mite spawning (if any).

How to reproduce your bug?:
Kill the infected trees in the area and the Mites should spawn under their corpses.

How it should work?:
It should spawn more Mites when they do spawn (the amount is written above and taken from the wowhead comments section for the patch we use).

Operating System:
Windows 10

**Comment by HEGladios** [comment]
Confirmed

**Comment by dikkedeur** [comment]
interesting, as i look in the script, since its odd this one npc's are scripted in c++ why not in smartai is a miracle to me, since it looks to me it can be done better in smartai...

**Comment by dikkedeur** [comment]
well well, this was not an easy one, the spell that is being used used to summon 4 wood mites, but it always summons 1. so then there is no other way then to hack fix this shit in smartai.
so what i did was remove the c++ script, then rescripted the npc that on death it has a change on at least cast 1 time the spell, so 1 wood mite would spawn ( on your location ) but it also has a 80 % to do it again, and 60% and 40% and 20%.. so hackfix, yeah, but it works, on death you have roughly 3 to 5 wood mites, it depends, but thats the beauty of it.. also that other protector has more change since i did it on 100% back to 10%, so it can be 1 to 9 roughly, also the root walkers will cast regrowth now and based on wowhead comments on a rather fast rate, so he is not easy anymore, als the rotten one will cast thunderclap now based on wowhead..

so in short, this was a pain, and took me hours to finally make it the way i wanted it to have..
but its worth it!

on next core update.. this will be active then
